spreadsheet_oca/security/ir.model.access.csv

```diff
@@ -4,3 +4,5 @@ access_spreadsheet_oca_revision,access_spreadsheet_oca_revision,model_spreadshee
 spreadsheet_oca.access_spreadsheet_spreadsheet_import,access_spreadsheet_spreadsheet_import,spreadsheet_oca.model_spreadsheet_spreadsheet_import,base.group_user,1,1,1,1
 access_spreadsheet_import_mode,access_spreadsheet_oca_revision,model_spreadsheet_spreadsheet_import_mode,base.group_user,1,0,0,0
 access_spreadsheet_select_row_number,access_spreadsheet_select_row_number,model_spreadsheet_select_row_number,base.group_user,1,1,1,1
+access_spreadsheet_spreadsheet_tag,access_spreadsheet_spreadsheet_tag,model_spreadsheet_spreadsheet_tag,spreadsheet_oca.group_user,1,0,1,0
```

Comment:
```suggestion
access_spreadsheet_spreadsheet_tag,access_spreadsheet_spreadsheet_tag,model_spreadsheet_spreadsheet_tag,spreadsheet_oca.group_user,1,1,1,0
```

At least. Otherwise, if a user create a tag, ans if he made a mistake in the name, he can not change it.

Reply:
What is your opinion here @flotho, and what do you need ?

Reply:
¿Then should it be this?:

```suggestion
access_spreadsheet_spreadsheet_tag,access_spreadsheet_spreadsheet_tag,model_spreadsheet_spreadsheet_tag,spreadsheet_oca.group_user,1,0,0,0
```